lxd/daemon.go

```diff
@@ -1321,12 +1329,19 @@ func (d *Daemon) init() error {
 	d.gateway.HeartbeatOfflineThreshold = d.globalConfig.OfflineThreshold()
 	lokiURL, lokiUsername, lokiPassword, lokiCACert, lokiLabels, lokiLoglevel, lokiTypes := d.globalConfig.LokiServer()
 	oidcIssuer, oidcClientID, oidcAudience := d.globalConfig.OIDCServer()
-
 	instancePlacementScriptlet := d.globalConfig.InstancesPlacementScriptlet()
 
 	d.endpoints.NetworkUpdateTrustedProxy(d.globalConfig.HTTPSTrustedProxy())
 	d.globalConfigMu.Unlock()
 
+	// Set OVN log monitor.
+	if ovnAddress != "" {
+		d.ovnLogMonitorCancelFunc, err = d.ovnLogMonitor.Start(d.shutdownCtx, ovnAddress)
```

Comment:
Returning a cancel function here isn't the normal way that one would use contexts, normally one would use `ctx , cancelFunc := context.WithCancel(d.shutdownCtx)` here and then pass that new local ctx into d.ovnLogMonitor.Start()`.